src/multi.rs

```diff
@@ -227,13 +225,16 @@ impl MultiState {
             free_set: vec![],
             ordering: vec![],
             draw_target,
-            move_cursor: false,
             alignment: MultiProgressAlignment::default(),
             orphan_lines: Vec::new(),
             zombie_lines_count: VisualLines::default(),
         }
     }
 
+    fn set_move_cursor(&mut self, move_cursor: bool) {
```

Comment:
Don't see much value in adding this method? Looks like `MultiProgress::set_move_cursor()` can just reach into `MultiState::draw_target` directly.

Reply:
You are right, fixed :+1: